src/control/KinematicSimulationTrajectoryExecutor.cpp

```diff
@@ -105,8 +105,9 @@ void KinematicSimulationTrajectoryExecutor::step()
     mInExecution = false;
   }
 
+  using seconds = std::chrono::duration<double>;
```

Comment:
This is a potentially confusing alias, because `std::chrono::seconds` stores integers and this alias stores `double`s.